build-manual

```diff
@@ -2,6 +2,7 @@
 
 python -m pip install -r manual_requirements.txt
 python -m pip install sphinx sphinx-book-theme sphinx-copybutton jupyter-sphinx
+python -m pip install qiskit-algorithms ~= 0.2.1
```

Comment:
Should this be a requirement of pyket-qiskit in the future?

Reply:
It's a one-line fix to pytket-qiskit that I have on a branch (where I'm trying to fix all the deprecation warnings). I can make a PR with just this fix.

Reply:
I don't think so.

This is for quantum algorithms so I think it contains things like pre-built circuits for Grover's algorithm etc.

I don't think this is within the scope of pytket-qiskit right now.

Its possible we may need to update the pytket-qiskit tests though as I think `qiskit.algorithms` is used there.